lwtime/nattime.go

```diff
@@ -178,13 +178,18 @@ func (nt natural) getRelativeRange() (relStart string, relEnd string, err error)
 }
 
 // Parse the string representation of a Lacework natural time
+// Start and End time objects are returned in UTC
 //
 // start, end, err := lwtime.ParseNatural("this year")
-// if err != nil {
-// 	...
-// }
+//
```

Comment:
not sure why this formatting changed.  likely auto formatting from vscode.  will put it back